ansible/vars/icds/icds_public.yml

```diff
@@ -175,7 +175,7 @@ etc_host_lines:
   - '10.247.24.13		www.icds-cas.gov.in'
 etc_host_lines_removed: ['etc_host_lines']
 
-http_proxy_address: '10.247.63.132'
+http_proxy_address: '10.247.24.16'
```

Comment:
@snopoke From what I understand all new machines will need to be tunneled through 24.16 until they are whitelisted on the patch server. There has been no resolution in our email. search for subject "additional VMs to patch server" @mkangia may know more, but they haven't responded for over a week on that thread